server/src/main/java/org/elasticsearch/index/IndexSettings.java

```diff
@@ -248,8 +248,7 @@
      * This setting will be ignored if soft-deletes is used in peer recoveries (default in 7.4).
      **/
     public static final Setting<TimeValue> INDEX_TRANSLOG_RETENTION_AGE_SETTING =
```

Comment:
should we deprecate these settings in 7 and disallow setting them on 8.x indices?

We should also add docs in this PR saying that these settings are no longer supported and are effectively ignored

Reply:
> should we deprecate these settings in 7 and disallow setting them on 8.x indices? We should also add docs in this PR saying that these settings are no longer supported and are effectively ignored


Yup, It was planned in the next step :)